src/validation.js

```diff
@@ -25,7 +25,7 @@ export class Validation {
    * @param subject The subject to validate
    * @returns {ValidationGroup} A ValidationGroup that encapsulates the validation rules and current validation state for this subject
    */
-  on(subject, configCallback) {
+  on(subject, configCallback?: any): ValidationGroup {
```

Comment:
it looks like correct type for configCallback is `(conf: ValidationConfig) => void`